src/main/modules/api/cos-api.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosInstance } from 'axios';
+import { LoggerInstance } from 'winston';
+
+export class COSApi {
+  constructor(
+    private readonly axios: AxiosInstance,
+    private readonly logger: LoggerInstance
+  ) { }
+
+  public createCase(data: any): Promise<boolean> {
+    return this.axios
+      .post('/submit', data)
+      .then(results => !!results)
+      .catch(err => {
+        this.logger.error(err);
+        throw new Error('Case could not be created. Please try again.');
+      });
```

Comment:
You could make this easier to read by making it async e.g.

```suggestion
      let results = false;
      try {
        results = !!(await this.axios.post('/submit', data));
      } catch (e) {
        this.logger.error(err);
        throw new Error('Case could not be created. Please try again.');
      }
      return results;
```

Reply:
imo i think the then and catch make is as easier to read plus looks nicer visually. Will discuss with sarah what is being returned so at that point, there might be difficulty there so the approach might be better but even then i think the then block can handle it.